### PR TITLE
M1 setup project

### PR DESCRIPTION
**Pull request by ahzia**
In this Milestone, I just Write plain HTML markup with minimum styling (just to make the layout).
I used Git Flow for managing Branches and I also Configured Webpack and linters

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly update the readme to match this [template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+Milstone 2: send and receive data from API.
+Milstone 3: final touches.
+
+## Built With
+
+- HTML, CSS, JS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (Dipeshtwis) [inline README.md]: - [x] I would invite you to update the `live link` or remove the section for now if you don't have any live link to make the README professional.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @ahzia, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Webpack is well configured.
:+1: Used ES6 modules to write modular JavaScript with proper ES6 syntax.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Status: approved🎉 🎉 

## Hi @ahzia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@mikenath223 , @Dipeshtwis , @DeliceLydia  Thanks for the Reviews

**Pull request merged**